### ISSUE TITLE
Deploy Frontend to Amazon S3

### ISSUE THREAD

**Issue by atasoyfurkan** [bounswe/2021SpringGroup5#492]
- Deploy the app to the Amazon server
- Build the code before deploying
- Amazon S3 will be used since Amazon Amplify have some issues with backend integration

**Comment by atasoyfurkan** [comment]
I will search on [this website](https://andela.com/insights/how-to-deploy-your-react-app-to-aws-s3/) and follow the instructions.

**Comment by atasoyfurkan** [comment]
I created a bucket named 'cmpe451-frontend' in Amazon S3 to be able to deploy application. Then, I created a user with admin privileges to edit the bucket. Then, I changed permission of the bucket to read only by all users and I enabled the static web hosting so the S3 server can be used by React App. 

Then, I downloaded amazon-cli to my local and I add deploy command to package.json in order to deploy react app to Amazon S3.

**Comment by atasoyfurkan** [comment]
After the step I said, the bucket was still not working. Then, I searched it from [this website](https://medium.com/@edtimmer/s3-bucket-set-up-must-have-changed-a-bit-and-as-a-result-i-run-into-a-significant-issue-where-i-618641966703) and I changed a couple of permissions so that the error is resolved. Now it is hosted successfully. It can be accesible by this link: http://cmpe451-frontend.s3-website.us-east-2.amazonaws.com
